package.json

```diff
@@ -89,6 +90,9 @@
     "vuetify-loader": "^1.3.0",
     "webpack": "^4.41.0"
   },
+  "resolutions": {
+    "istanbul-instrumenter-loader/**/istanbul-lib-instrument": "4.0.1"
+  },
```

Comment:
Overriding the dependency of the loader, as the new Istanbul instrument library loads Babel plug-ins. Alas it seems like a release with the new version might take a while over there.